### PR TITLE
feat: cs 43-13 receive project creation notification

### PR DESCRIPTION
**Pull request by salet-gutierrez-jalafund**
## Related Tickets & US
- [[CS-43-13] Receive project creation notification](https://app.clickup.com/t/868b2kd4k)

## Description

A class called `CoverageNotificationController` has been added which will help to update any coverage changes from ArgosUI and save them to the database.

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

It can be tested by going to the `CoverageNotificationControllerTest` test which will show the test cases for the controller.

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I have checked my code and corrected any misspellings

- - -
This change is [Reviewable](https://reviewable.io/reviews/SoftDevSix/ArgosAPI/24)

### CODE REVIEW

File: src/main/java/com/softdevsix/api/controllers/CoverageNotificationController.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/coverageNotification")
```
Comment (zrock-dev) [inline src/main/java/com/softdevsix/api/controllers/CoverageNotificationController.java]: Please rename endpoint to coverage/creation-notification. To follow the documentation and reduce issues.

**Review by zrock-dev — CHANGES_REQUESTED** [review]
Good job just one small change

**Review by ronaldo-mendoza-jalafund — APPROVED** [review]
nice job

File: src/main/java/com/softdevsix/api/controllers/CoverageNotificationController.java (hunk 1 above)
Comment (zrock-dev) [reply src/main/java/com/softdevsix/api/controllers/CoverageNotificationController.java]: Done. 
signed by Salet

**Review by zrock-dev — APPROVED** [review]
LGTM

**Pull request merged**